If a cover profile is empty, combining them crashes

# What happens

When I run `ginkgo -r -randomizeAllSpecs -randomizeSuites -failOnPending -cover -trace -race -progress -coverprofile=coverage.out -outputdir=.`, I get :
```
panic: runtime error: index out of range [-1]

goroutine 1 [running]:
main.(*SpecRunner).combineCoverprofiles(0xc000184000, 0xc0001bc100, 0x18, 0x20, 0xc000110098, 0x0)
	/home/sebastien/Code/go/src/github.com/onsi/ginkgo/ginkgo/run_command.go:196 +0x7b1
main.(*SpecRunner).RunSpecs(0xc000184000, 0xc0001bc0a0, 0x0, 0x6, 0xc906e0, 0x0, 0x0)
	/home/sebastien/Code/go/src/github.com/onsi/ginkgo/ginkgo/run_command.go:116 +0xb66
main.(*Command).Run(0xc000186000, 0xc0001bc000, 0xa, 0x10, 0xc906e0, 0x0, 0x0)
	/home/sebastien/Code/go/src/github.com/onsi/ginkgo/ginkgo/main.go:158 +0xb8
main.main()
	/home/sebastien/Code/go/src/github.com/onsi/ginkgo/ginkgo/main.go:206 +0x13c
```

# Something more or less specific to my situation

I use a script to create test suites in all directories containing Go files (thus, in all packages of a given repository). Some of these packages only contain struct definitions with no function, so no statement to test.

Basicly, no problem with such a suite, it just says "coverage: [no statements]" and writes only the cover mode line into the file...

# The problem

... but if I want to combine all coverage results (with `-outputdir=.`), Ginkgo removes the cover mode line and then tries to add a `\n` at the end of all contents.

Code is here: https://github.com/onsi/ginkgo/blob/4ba33d41e7e3168d83148303403b5a5d6682ef58/ginkgo/run_command.go#L196

Problem: without the cover mode line and without any statement, len(contents) equals 0 and `contents[len(contents)-1]` results in the aforementioned panic.

--- a/ginkgo/run_command.go
+++ b/ginkgo/run_command.go
@@ -193,7 +193,7 @@ func (r *SpecRunner) combineCoverprofiles(runners []*testrunner.TestRunner) erro
 		_, err = combined.Write(contents)
 
 		// Add a newline to the end of every file if missing.
-		if err == nil && contents[len(contents)-1] != '\n' {
+		if err == nil && len(contents) > 0 && contents[len(contents)-1] != '\n' {
 			_, err = combined.Write([]byte("\n"))
 		}
 

